app.js

```diff
@@ -13,7 +13,9 @@ const http = require('http')
 	, serveStatic = require('serve-static')
 	, compression = require('compression')
 	, phantomjs = require('phantomjs-prebuilt')
-	, logoScrape = require('logo-scrape');
+	, logoScrape = require('logo-scrape')
+  , chrome = require('selenium-webdriver/chrome')
+  , { Builder } = require('selenium-webdriver');
```

Comment:
Could we Prettier this file too?

Reply:
It would be a lot of changes in this file. I think we could refactor this whole file and get rid of a lot of the noise.